Makefile

```
@@ -5,7 +5,7 @@ init:
 	pip install -Ur requirements.txt
 
 build:
-	build-image-src/build_all_images.sh
+	cd build-image-src; ./build_all_images.sh
```

So that the build isn't executed if `cd` fails:

```suggestion
	cd build-image-src && ./build_all_images.sh
```